SPECS/libnotify/libnotify.spec

```diff
@@ -38,10 +37,10 @@ development of programs using %{name}.
 
 %prep
 %setup -q
-%patch0 -p1
+%patch 0 -p1
 
 %build
-%meson -Dman=false
+%meson -Dman=false -Dgtk_doc=false -Dtests=false
```

Comment:
why do we want to disable doc generation and build tests?

Reply:
gtk-doc requires network access during build, which we don't have.
Tests may not be required during the build. (so is it in Fedora as well)
Ref:
* Wed Mar 03 2021 Henry Li <lihl@microsoft.com> - 0.7.9-2
- Initial CBL-Mariner import from Fedora 32 (license: MIT).
- Remove gtk dependency
- Apply patch to remove gtk dependency and disable tests, which depends on gtk;
  Disable gtk-doc and docbook, which requires network access for downloading files

Reply:
I moved the disabling of meson options from patch to the SPEC to make the maintenance simple

Reply:
when you specify these options in meson command then patch won't be needed.